fastapi/openapi/utils.py

```diff
@@ -126,6 +126,9 @@ def get_openapi_operation_request_body(
     if required:
         request_body_oai["required"] = required
     request_body_oai["content"] = {request_media_type: {"schema": body_schema}}
+    examples = body_schema.get("examples")
```

Comment:
would you like to add some unit tests for this code in your PR?

Reply:
Yeah, i'd be happy to. I have waited to add tests and docs because I wanted to get some feedback on #822 and the proposed solution in this PR from the library maintainers, but if this looks good to you and to @tiangolo (I'm tagging him here because he had commented on the issue) then I'd be glad to add tests and even a little docs demo of the functionality.

Reply:
Rebased, added tests and a brief edit to the docs. I also added a screen grab, but my system is slightly different from the system used to create other `/docs` screenshots.